mk/config.mk

```diff
@@ -205,6 +205,21 @@ CFG_WITH_USER_TA ?= y
 # By default, in-tree TAs are built using the first architecture specified in
 # $(ta-targets).
 
+# Address Space Layout Randomization for user-mode Trusted Applications
+#
+# When this flag is enabled, the ELF loader will introduce a random offset
+# when mapping the application in user space. ASLR makes the exploitation of
+# memory corruption vulnerabilities more difficult.
+CFG_TA_ASLR ?= n
```

Comment:
The config used by HiKey could be applicable here as generic:
```Patch
-CFG_TA_ASLR ?= n
+ifneq ($(CFG_WITH_PAGER),y)
+CFG_TA_ASLR ?= y
+endif
```

Reply:
My latest tests show that `CFG_TA_ASLR=y` is fine for HiKey960 with or without pager. I have updated the config accordingly.